src/http.cc

```diff
@@ -661,14 +662,13 @@ HttpStateData::processReplyHeader()
     /** Creates a blank header. If this routine is made incremental, this will not do */
 
     /* NP: all exit points to this function MUST call ctx_exit(ctx) */
-    Ctx ctx = ctx_enter(entry->mem_obj->urlXXX());
+    DebugContext ctx(entry->mem_obj->urlXXX());
```

Comment:
Why the mix between "DebugContext ctx(..." and "auto ctx = DebugContext(..." ?

Please try to make them all use one style if possible and if you have to pick the second, document why in the PR description.

Reply:
Switching all to the first form